index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="style.css" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>project1_yeaeunmoon</title>
```

Comment:
The `title` element's content can be normal text and include spaces. It should generally be human-readable (since it's what users see when they bookmark a page, for example). So this would be valid:
```suggestion
    <title>Project 1 Yeaeun Moon</title>
```